README.md

```diff
@@ -65,14 +65,16 @@ kubectl apply -f https://github.com/keptn/lifecycle-toolkit/releases/download/v0
 <!---x-release-please-end-->
 
 The Lifecycle Toolkit uses the OpenTelemetry collector to provide a vendor-agnostic implementation of how to receive,
-process and export telemetry data. To install it, follow
+process and export telemetry data.
+To install it, follow
 their [installation instructions](https://opentelemetry.io/docs/collector/getting-started/).
 We provide some information about this in our [observability example](./examples/support/observability/).
 
 The Lifecycle Toolkit includes a Mutating Webhook which requires TLS certificates to be mounted as a volume in its pod.
 The certificate creation
 is handled automatically
-by [klt-cert-manager](https://github.com/keptn/lifecycle-toolkit/blob/main/klt-cert-manager/README.md). Versions 0.5.0
+by [klt-cert-manager](https://github.com/keptn/lifecycle-toolkit/blob/main/klt-cert-manager/README.md).
+Versions 0.5.0
```

Comment:
this is actually valid for 0.6.0 since we postponed the release of the helm chart, would you mind correcting it ?